ambry-messageformat/src/main/java/com/github/ambry/messageformat/MessageFormatSend.java

```diff
@@ -104,9 +104,8 @@ public MessageFormatSend(MessageReadSet readSet, MessageFormatFlags flag, Messag
   /**
    * Calculates the offsets from the MessageReadSet that needs to be sent over the network
    * based on the type of data requested as indicated by the flags
-   * @param enableDataPrefetch do data prefetch if this is true.
    */
-  private void calculateOffsets(boolean enableDataPrefetch) throws IOException, MessageFormatException {
+  private void calculateOffsets() throws IOException, MessageFormatException {
```

Comment:
we could rename this method to `fetchDataFromDisk`, since that is its primary job now.

Reply:
+1